roles/maintenance_15_rhel/tasks/main.yml

```diff
@@ -33,21 +33,20 @@
   vars:
     taskid: 15-011
     name: "Does yum check-update work"
-  args:
-    warn: false
   ansible.builtin.command: "yum check-update -q"
   register: yum_checkupdate_output
   check_mode: no
+  failed_when: no
   changed_when: no
-  failed_when: "yum_checkupdate_output.stderr_lines | length > 0"
 
 - <<: *task
   vars:
     taskid: 15-011
-    name: "Does yum check-update work | Print new pkgs"
+    name: "Does yum check-update work | Show exit status"
   ansible.builtin.debug:
-    var: yum_checkupdate_output.stdout_lines
-  changed_when: no
+    msg: "Exited with: {{ yum_checkupdate_output.rc }}, Error is: {{ yum_checkupdate_output.stderr_lines|default('Not available') }}"
+  changed_when: "yum_checkupdate_output.rc != 0 and yum_checkupdate_output.rc != 100"
+  when: "yum_checkupdate_output.rc != 0 and yum_checkupdate_output.rc != 100"
```

Comment:
Having `when` and `changed_when` with the same condition confuses me a little. Can you clarify what's going on here?

On another note, the condition could be written shorter (and imo more understandable) as `yum_checkupdate_output.rc not in [0, 100]`

Reply:
For some reason I initially only wanted to show it when the status is neither 0/100 and in that case also have it be changed. But it's actually OK to just always show it (and with the default `ansible.cfg` being hidden). So it's only the `changed_when` now :)